app/routers/event.py

```diff
@@ -109,18 +110,30 @@ async def create_new_event(request: Request,
 
     messages = get_messages(session, event, uninvited_contacts)
     return RedirectResponse(router.url_path_for('eventview', event_id=event.id)
-                            + f'messages={"---".join(messages)}',
+                            + f'?messages={"---".join(messages)}',
                             status_code=status.HTTP_302_FOUND)
 
 
+def get_waze_link(event: Event) -> str:
+    # if event.latitude and event.longitude:
+    #     coordinates = f"{event.latitude},{event.longitude}"
+    #     return f"https://waze.com/ul?ll={coordinates}&navigate=yes"
```

Comment:
delete?

Reply:
No.  It is based on data waiting to enter the code